api/src/test/resources/application.yaml

```diff
@@ -0,0 +1,32 @@
+uk:
```

Comment:
Is it possible to not start spring for unit tests? I always feel that's the boundary with integration tests

Reply:
I had refactored the tests so that we don't need to start spring with unit tests but forgot to remove app config : )